cypress/component/ui/accordion/Accordion.spec.tsx

```diff
@@ -0,0 +1,29 @@
+import { Accordion } from '../../../../src/sections/ui/accordion/Accordion'
+
+describe('Accordion', () => {
+  it('renders an accordion with defaultActiveKey prop and children', () => {
```

Comment:
I think we should add a test 
1. to test the functionality of show/hide on click
2. to test the alwaysOpen prop